examples/stimuli/tracker_dealer.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+"""
+======================
+Tracker Dealer Example
+======================
+
+This file shows how to interleave multiple Tracker objects using
+:class:`expyfun.stimuli.TrackerDealer`.
+
+In this case, a modeled human subject generates two curves (one for each trial
+type: 1 & 2)
+
+@author: maddycapp27
+"""
+
+import numpy as np
+from expyfun.stimuli import TrackerUD, TrackerDealer
+from expyfun.analyze import sigmoid
+import matplotlib.pyplot as plt
+
+# define parameters of modeled subject (using sigmoid probability)
+true_thresh = [30, 40]
+slope = 0.1
+chance = 0.5
+
+# define parameters for each tracker (assuming each tracker uses same rules)
+up = 1
+down = 1
+step_size_up = [9, 3]
+step_size_down = [3, 1]
+stop_criterion = 30
+stop_rule = 'reversals'
+start_value = 45
+change_criteria = [0, 5]
+change_rule = 'reversals'
+x_min = 0
+x_max = 90
+
+# parameters for the tracker dealer
+max_lag = 2
+rand = None

```

Comment:
I think @Eric89GXL usually likes this to be called ``rng``. And I think in this case it is better to seed it, so ``rng = np.random.RandomState(0)``. Mess around with the seed value until you get plots that end up with thresholds within 1 or 2 of their true value. Seeding the RNGs will make the plots the same every time, and having the computed thresholds close to the the true ones in the example will be less confusing for people.